samples/README.md

```diff
@@ -0,0 +1,63 @@
+<!--
+   samplefwlink:  https://aka.ms/WinIoTSamples
+--->
+
+# EFLOW Internet of Things (IoT) Samples
+
+This repo contains samples that demonstrate usage patterns for interoperatibility between Microsoft's Windows 10 and the EFLOW VM.  These code samples were created with templates available in Visual Studio and are designed, but not limited to, run on devices using EFLOW. For more information, review [EFLOW docs](https://docs.microsoft.com/en-us/windows/iot/iot-enterprise/eflow).
+
+> **Note:** If you are unfamiliar with Git and GitHub, you can download the entire collection as a 
+> [ZIP file](https://github.com/Microsoft/Windows-universal-samples/archive/master.zip), but be 
+> sure to unzip everything to access shared dependencies. For more info on working with the ZIP file, 
+> the samples collection, and GitHub, see [Get the UWP samples from GitHub](https://aka.ms/ovu2uq). 
+> For more samples, see the [Samples portal](https://aka.ms/winsamples) on the Windows Dev Center. 
+
+## Windows 10 development
+These samples require Visual Studio and the Windows Software Development Kit (SDK) for Windows 10.
+
+   [Get a free copy of Visual Studio Community Edition with support for building Universal Windows Platform apps](http://go.microsoft.com/fwlink/p/?LinkID=280676)
+
+Additionally, to stay on top of the latest updates to Windows and the development tools, become a Windows Insider by joining the Windows Insider Program.
+
+   [Become a Windows Insider](https://insider.windows.com/)
+
+## Samples by category
+
+### Azure IoT Edge
```

Comment:
For categories, lets do something a little more granular since we are already in Azure IoT Edge.  I would suggest something like "Interop" and "Hardware Access".  Just an example, if you have a better Idea that is great too.

Reply:
Yes great - will separate and add Camera passthrough to the Hardware access part